_inc/client/at-a-glance/backups.jsx

```diff
@@ -50,8 +50,8 @@ const DashBackups = React.createClass( {
 
 						<p className="jp-dash-item__description">
 							{ vpData.message }
-							<br/>
-							{ __( '{{a}}View backup details{{/a}}', {
+
+							{ __( ' {{a}}View backup details{{/a}}.', {
```

Comment:
Just wondering if this space before the curly brackets `' {{a` is intentional. If it's for spacing purposes, it may be replaced by a `&nbsp;` before the JS code block